internal/pkg/api/server/templates/service/mapper.go

```diff
@@ -190,7 +190,7 @@ func InputsResponse(stepsGroups input.StepsGroupsExt) (out *Inputs) {
 					Options:     OptionsResponse(in.Options),
 				}
 				if in.ComponentId != "" {
-					inputResponse.ComponentID = &in.ComponentId
+					inputResponse.ComponentID = in.ComponentId
```

Comment:
jak jsem psal výš, `in.ComponentId` už není pointer na string ale string, tak se nemusí dereferencovat.